### PR TITLE
Update Reaper's documentation for v2

### LINKED ISSUE #1368: K8SSAND-1339 ⁃ Doc: Tasks > Repair topic needs review and updates for K8ssandra Operator multi-cluster
**What happened?**

The "Repair Cassandra with Reaper" topic at https://docs-v2.k8ssandra.io/tasks/manage/private-registries/ is just a copy of the v1.4.x topic.  Need a SW engineer to review and update the content for K8ssandra Operator multi-cluster. 

**How to reproduce it (as minimally and precisely as possible)**:

* https://github.com/k8ssandra/k8ssandra/blob/docs-v2/docs/content/en/tasks/repair/_index.md
* https://docs-v2.k8ssandra.io/tasks/repair/

**K8ssandra Operator version**
1.0.x.

**Anything else we need to know?**:

* Revise the topic on the `doc-v2` branch of k8ssandra/k8ssandra GH repo. 
* When done and PR is merged, please tag and publish the updates. Tag & publish steps: https://github.com/k8ssandra/k8ssandra/tree/docs#github-actions-and-tags

┆Issue is synchronized with this [Jira Task](https://k8ssandra.atlassian.net/browse/K8SSAND-1339) by [Unito](https://www.unito.io)
┆friendlyId: K8SSAND-1339
┆priority: Medium

### PR DESCRIPTION
**Pull request by adejanovski**
**What this PR does**:
Updates Reaper documentation for v2 (k8ssandra-operator).

**Which issue(s) this PR fixes**:
Fixes #1368 

**Checklist**
- [ ] Changes manually tested
- [ ] Automated Tests added/updated
- [x] Documentation added/updated
- [ ] CHANGELOG.md updated (not required for documentation PRs)
- [x] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: docs/content/en/tasks/repair/_index.md [hunk 1]
```diff
@@ -7,27 +7,26 @@ description: "Use the Reaper for Apache Cassandra&reg; web interface to perform
...
 
-{{% alert title="Tip" color="success" %}}
-As an alternative to configuring an Ingress, consider **port forwarding**. It's another way to provide external access to resources that have been deployed by K8ssandra Operator in your Kubernetes environment. Those resources could include Prometheus metrics, pre-configured Grafana dashboards, and the Reaper web interface for repairs of Cassandra data. The `kubectl port-forward` command does not require an Ingress/Traefik to work. 
+Access to the Reaper web interface requires either:  
 
-* Developers, see [Set up port forwarding]({{< relref "/quickstarts/developer/#set-up-port-forwarding" >}}).  
-* Site reliability engineers, see [Configure port forwarding]({{< relref "/quickstarts/site-reliability-engineer/#port-forwarding" >}}).
-{{% /alert %}}
+* setting up a custom Ingress resource
+* or modifying Reaper's Kubernetes service into a LoadBalancer (in cloud environments), which will expose it over a public IP
```
Comment (jeffbanks) [inline docs/content/en/tasks/repair/_index.md:26]: Suggestion - "or modifying Reaper's Kubernetes service to function **as a** LoadBalance (in cloud environments), ... "

File: docs/content/en/tasks/repair/_index.md [hunk 2]
```diff
@@ -56,23 +45,22 @@ kubectl get pods
...
+k8ssandra-operator-6d4dd9fb8f-5kzl6                          1/1     Running   0          10d
 ```
 
 ## What can I do in Reaper?
 
 To access Reaper, navigate to [http://localhost:8080/webui/](http://localhost:8080/webui/). 
 
 {{% alert title="Tip" color="success" %}}
-Again, if you are not running locally, use the hostname value in the URL that you specified on the `--set reaper.ingress.host` flag above.
+Again, if you are not running locally, use the IP address provided by your cluster for the ingress resource or the load balancer service.
```
Comment (jeffbanks) [inline docs/content/en/tasks/repair/_index.md:63]: nit - could remove "Again, "

**Review by jeffbanks — APPROVED** [review]
Look good, just a couple of minor suggestions.

File: docs/content/en/tasks/repair/_index.md (hunk 1 above)
Comment (adejanovski) [reply docs/content/en/tasks/repair/_index.md]: I made the change 👍

File: docs/content/en/tasks/repair/_index.md (hunk 2 above)
Comment (adejanovski) [reply docs/content/en/tasks/repair/_index.md]: Done 👍

**Pull request merged**